webpack.config.js

```diff
@@ -51,6 +51,13 @@ module.exports = {
           {
             loader: 'css-loader'
           },
+          {
+            loader: 'postcss-loader',
+            options: { plugins: [
+              require('autoprefixer')({ grid: true })
```

Comment:
I like the autoprefix + "grid: true" option
Another good intermediary for v8 :)